### PR TITLE
Add a property for Permissions class : Scopes 

### PR DESCRIPTION
**Pull request by VinceGusmini**
Scopes is a Dictionary<string, Scope>.
I also added a Scope class that just contains a property "status" (string).

### CODE REVIEW

**Comment by stoiveyp** [comment]
Hi @VinceGusmini - thank you for the contribution. Would you mind linking to the specs on this please?

I'm seeing consentToken in my Alexa requests in the console, but not the scopes object you're adding. Also I can't see it in the request schema:

https://developer.amazon.com/docs/custom-skills/request-and-response-json-reference.html#session-object

Be really useful just so we can make sure the structure you've got matches what's expected. Thank you.

**Comment by VinceGusmini** [comment]
Hi @stoiveyp ! 

Here's the a piece of a [post](https://alexa.uservoice.com/forums/906892-alexa-skills-developer-voice-and-vote/suggestions/36665794-add-additional-permission-scopes-to-alexa-response): 
"Currently an Alexa response includes permission scopes for "payments:autopay_consent" and "alexa::devices:all:geolocation:read" if the Skill manifest requests those permissions. The status will either be "GRANTED" or "DENIED". If "DENIED" the Skill can request the user enable the permission."

In summary I need to check if **context.System.user.permissions.scopes['alexa::devices:all:geolocation:read'].status** is **GRANTED**.

You can also see this doc "detect-which-specific-permissions-are-needed": 
https://developer.amazon.com/fr/docs/custom-skills/location-services-for-alexa-skills.html#detect-which-specific-permissions-are-needed

I hope this will answer to your question 👍

**Comment by stoiveyp** [comment]
Thanks @VinceGusmini, really appreciate it - it was in my location service skill, good catch!

The change itself looks spot on, could we please update the test library so we can maintain this moving forward, then it's just for Tim to merge straight in :+1:

File: Alexa.NET/Request/Scope.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+
+namespace Alexa.NET.Request
+{
+    public class Scope
+    {
+        [JsonProperty("status")]
+        public string status { get; set; }
```
Comment (timheuer) [inline Alexa.NET/Request/Scope.cs]: Just to be consistent with other classes can we make the public property 'Status' (capital) (keeping the JsonProperty as-is.

**Comment by timheuer** [comment]
@VinceGusmini I made one comment and yes if you could add a test case for this, I'll merge!  Thank you!

File: Alexa.NET/Request/Scope.cs (hunk 1 above)
Comment (VinceGusmini) [reply Alexa.NET/Request/Scope.cs]: Yes ! My bad sorry.

**Comment by timheuer** [comment]
Merged
